azure-pipelines.yml

```diff
@@ -23,6 +23,42 @@ steps:
   clean: true
   fetchDepth: 1
 
+- script: |
+    printfuncs="Print,Printf,Info,Infof,Warning,Warningf,Error,Errorf,Critical,Criticalf,Sprint,Sprintf,Log,Logf,Panic,Panicf,Fatal,Fatalf,Notice,Noticef,Wrap,Wrapf,WithMessage"
```

Comment:
The printfuncs were related to the logging package used by fabric and should not apply here. The default list is sufficient.